tests/test_core.py

```diff
@@ -5373,10 +5389,14 @@ def test_fs_append(self, js_engines=None):
 
   def test_fs_mmap(self):
     orig_compiler_opts = self.emcc_args[:]
+    create_test_file('pre.js', '''
+      // ignore leaked mmaps
```

Comment:
Why though?

Reply:
Use `self.add_pre_run('<literal>')` for this?

Reply:
I tried it this way to avoid intrusive changes to the test (which is not trivial).

However, it passes for me now without it, so I'm not sure why it was necessary before. Perhaps I had leak checks forced on somehow, though I don't remember that. Anyhow, removing this.